Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -1096,14 +1100,18 @@
   id: HeadofSecurityOuterClothing
   name: loadout-group-head-of-security-outerclothing
   loadouts:
-  - HeadofSecurityCoat
-  - HeadofSecurityWinterCoat
+#  - HeadofSecurityCoat # DeltaV - removed for incongruence
+#  - HeadofSecurityWinterCoat # DeltaV - removed for incongruence
+  - PlateCarrier # DeltaV
+  - DuraVest # DeltaV
```

Comment:
Why would a HoS use these, warden has advanced plate, and if these new HoS clothes are just better, these will be useless.

Reply:
Advanced plate is in the locker which is suuuuuch a hike (/j), sometimes there's bureaucratic errors and the only adv. plate carrier is in use, or uh. Edge cases, niche scenarios. The odds are that they just won't use em, but it doesn't necessarily hurt a bunch to give the option.